### ISSUE TITLE
3.4 release (web-ext) not working in Firefox mobile

### ISSUE THREAD

**Issue by dhowe** [dhowe/AdNauseam#1212]
Clicking toolbar button has no effect:

![image](https://user-images.githubusercontent.com/737638/30781826-c8cec2e0-a158-11e7-961d-b3dee2e6453a.png)

**Comment by Atavic** [comment]
Maybe related to https://github.com/gorhill/uBlock/issues/2958 ?

**Comment by dhowe** [comment]
Yes, quite probably. As a workaround, I've re-enabled our legacy version 1.4.103 for users prior to Firefox 57. Then (hopefully) we can resolve this before our next release (v1.5)

**Comment by mneunomne** [comment]
Outdated. Closing.
